packages/tenderly-core/src/internal/core/services/TenderlyApiService.ts

```diff
@@ -1,35 +1,59 @@
 import * as axios from "axios";
 
+import { logger } from "../../../utils/logger";
 import { getConfig, isAccessTokenSet } from "../../../utils/config";
 import { TENDERLY_API_BASE_URL, TENDERLY_JSON_RPC_BASE_URL } from "../../../common/constants";
 
 export class TenderlyApiService {
   public static configureInstance(): axios.AxiosInstance {
+    logger.debug("Configuring instance...");
+
     const tdlyConfig = getConfig();
-    return axios.default.create({
+    const params = {
+      baseURL: TENDERLY_API_BASE_URL,
+      accessKey: tdlyConfig.access_key,
+    };
+
+    logger.debug("Configured instance with parameters:", {
       baseURL: TENDERLY_API_BASE_URL,
-      headers: { "x-access-key": tdlyConfig.access_key },
+      access_key:
+        tdlyConfig.access_key !== undefined && tdlyConfig.access_key !== null && tdlyConfig.access_key !== ""
```

Comment:
Same as above we cannot log user access_key

Reply:
I guess we don't want to log access_token values, so I think we have to make sure we are not doing this, it's not a question about the log value, but security and compliance I think.
In short, make sure we are **not** logging access token value in any place, not just this one. Does this make sense?